conf/machine/imx6qdlsabreauto.conf

```diff
@@ -42,4 +42,4 @@ SERIAL_CONSOLES = "115200;ttymxc3"
 
 MACHINE_FIRMWARE_append_mx6 = " linux-firmware-ath6k"
 
-MACHINE_FEATURES += " pci wifi bluetooth"
+MACHINE_FEATURES += "pci wifi bluetooth optee"
```

Comment:
Please move the commit which enables the `optee` as the last commit so it does not breaks the tree on the commits in the PR.